.github/workflows/release.yml

```diff
@@ -59,6 +34,9 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Optionally, set artifact qualifier (for private builds that include Kakadu)
+        if: matrix.build_property == 'kakadu.version'
+        run: echo "ARTIFACT_QUALIFIER=-ucla" >> $GITHUB_ENV
```

Comment:
Maybe it's better to use Maven profiles to construct the image identifier, since this has to be repeated in each workflow now. Thoughts?

Reply:
It does sound like profiles would allow for less repetition.